src/test/java/omero/model/utests/UnitsTest.java

```diff
@@ -90,4 +90,14 @@ public void testLengthMappingFromMeterToCentimeter() throws BigResult {
     public void testLengthMappingFromCentimeterToMeter() throws BigResult {
         new LengthI(mm(1, UnitsLength.CENTIMETER), UnitsLength.METER);
     }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testLengthMappingFromPixelToMicrometer() throws BigResult {
```

Comment:
Based on https://github.com/ome/omero-blitz/pull/76/files#diff-9482bd905d1ca57fc58a7c0ab5c24c461b67925741607c88094166767eb68259R1460, I would have expected this to fail with an `IllegalArgumentException` rather than `BigResult` in this scenario

Reply:
?

It does.

Reply:
... don't mind me, I read the `throws BigResult` rather than `exceptedExceptions`.

I'll have my cup of coffee before commenting on a Monday morning next time